autowisp/browser_interface/results/lightcurve_views.py

```diff
@@ -380,7 +380,7 @@ def _subdivide_figure(plot_config, new_splits, current_splits, children):
             _subdivide_figure(plot_config, new_splits, *child)
 
 
-def update_subplot(plotting_session, updates):
+def update_subplot(plotting_session, updates, project_home=None):
```

Comment:
You don't need to pass `project_home` as argumnent. You can get it from `ImageProcessingManager.project_home`, since `ImageProcessingManager` inherits `ProcessingManager` which must have the home set in order for anything to work.